qt/applications/workbench/workbench/plotting/figuremanager.py

```diff
@@ -510,8 +510,6 @@ def waterfall_reverse_line_order(self):
             for line_fill in fills:
```

Comment:
I wonder if we can just remove the version checks here and replace with a comment. Presumably the code will break in 3.7 anyway so we don't need to raise our own exception.

Reply:
I'm amenable to that. Will also raise an issue to deal with waterfall plots as v3.7 appears to be out already so this problem will be a problem sooner rather than later.